docs/advanced/quickstart-split.md

```diff
@@ -96,3 +91,14 @@ Created Charon cluster:
 ```
 
 These split keys can now be used to start a Charon cluster.
+
+## Step 3. (Optional) Encrypt artifacts for distribution
+
+Within each folder are the encrypted [private key shares](../int/key-concepts.md#distributed-validator-key-share), along with the decryption passwords. To transmit these folders to the operators/machines where they will run, it might be prudent to encrypt the folder as a `.zip` to transport them.
```

Comment:
```suggestion
Within each folder are the encrypted [private key shares](../int/key-concepts.md#distributed-validator-key-share), along with the decryption passwords. To transmit these folders to the operators/machines where they will run, it might be prudent to compress the folder to a `.zip` to transport them.
```

Might be better word for zipping... we are not really encrypting when we make it a zip.

Reply:
The `-e` flag does encrypt it, the command prompts you to enter a password. :)